src/wrappers/vue-wrapper.js

```diff
@@ -24,9 +23,9 @@ export default class VueWrapper extends Wrapper implements BaseWrapper {
     }))
     this.vm = vm
     this.isVueComponent = true
-    this._emitted = Object.create(null)
-    this._emittedByOrder = []
+    this._emitted = options.eventCaptor.emitted
+    this._emittedByOrder = options.eventCaptor.emittedByOrder
```

Comment:
We should add this._emitted (maybe this.__emitted to be safe) in the mixin, then every component will have them added. Could you write a test to make sure this is true? Every child component should have a clean emitted object that only refers to events emitted by itself